jobs/dap-collector-ppa-prod/dap_collector_ppa_prod/main.py

```diff
@@ -208,14 +222,55 @@ async def collect_many(
 
 
 async def collect_task(task, auth_token, hpke_private_key, date):
-    """Collects data for the given task and the given day."""
-    start = datetime.datetime.fromisoformat(date)
-    start = start.replace(tzinfo=datetime.timezone.utc)
-    end = start + datetime.timedelta(minutes=JOB_INTERVAL)
+    """Collects data for the given task through to the given day.
+        For tasks with time precision smaller than a day, will collect data for aggregations from the day prior to date.
+        For tasks with time precision a day or multiple of day, will collect data for the aggregation that ends on date.
+            If date does not align with the end of an aggregation, it will not collect anything.
+    """
+    until = datetime.datetime.fromisoformat(date)
```

Comment:
It might make sense to keep this variable name as `end` or `end_collection_date`, similarly it might make sense to keep until_minus_{x} labelled as `start` (or some variant).